persistence/sql/sqlsaga_required-in-some-versions_sqlpersistence_[4,5).partial.md

```diff
@@ -0,0 +1 @@
+INFO: In order to use the normal `Saga<T>` class for sagas, upgrade to at least SQL Persistence 4.2.
```

Comment:
> upgrade to at least SQL Persistence 4.2.

upgrade SQL Persistence to Version 4.2 or above or use SQL Persistence Version 4.2 or above

Reply:
I made this change. I don't know why GitHub is still showing it this way!